cmd/cosign/cli/options/signblob.go

```diff
@@ -82,4 +83,7 @@ func (o *SignBlobOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&o.RFC3161TimestampPath, "rfc3161-timestamp", "",
 		"write the RFC3161 timestamp to a file")
 	_ = cmd.Flags().SetAnnotation("rfc3161-timestamp", cobra.BashCompFilenameExt, []string{})
+
+	cmd.Flags().BoolVar(&o.IssueCertificate, "issue-certificate", false,
+		"when set and a key is provided, issue a code signing certificate from Fulcio")
```

Comment:
This is a little odd as a description because when a key is *not* provided, we issue a code signing certificate from Fulcio.

Maybe:

> issue a code signing certificate from Fulcio, even if a key is a provided